hr_expense_hide_taxes/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Hr Expense Hide Taxes",
+    "summary": """
+        Hide the 'tax_ids' fields
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "17.0.1.0.0",
+    "category": "Operations/HR",
+    "website": "https://github.com/solvosci/slv-hr",
+    "depends": [
+        "hr_expense"
+    ],
+    "data": [
+        "views/hr_extense_view.xml",
```

Comment:
Fix typo and use views in plural, as usual:
```suggestion
        "views/hr_expense_views.xml",
```

Reply:
Still missing this